runtime/src/main/java/com/redhat/ipaas/rest/BaseHandler.java

```diff
@@ -0,0 +1,15 @@
+package com.redhat.ipaas.rest;
+
+import javax.inject.Inject;
+
+public class BaseHandler implements WithDataManager {
+
+    @Inject
+    private DataManager dataMgr;
+
+    @Override
+    public DataManager dataManager() {
```

Comment:
No that's fine - also the swagger generation requires getters, but that doesn't mean you have to use them when you don't need to. Or maybe I'm just being too radical, been corrupted by too many other languages now.